app/models/petition.rb

```diff
@@ -105,6 +107,23 @@ class Petition < ActiveRecord::Base
     where("signature_count >= ?", SystemSetting.value_of_key(SystemSetting::GET_AN_MP_SIGNATURE_COUNT).to_i)
   }
 
+  def sponsor_emails
+    @sponsor_emails || []
+  end
+
+  def sponsor_emails=(emails)
```

Comment:
I'd consider making this method assume the emails is an array already and do the string->array massaging in the controller.